packages/launchpad-ledger/src/ledgersigner.ts

```diff
@@ -1,22 +1,28 @@
 import { HdPath } from "@cosmjs/crypto";
 import {
   AccountData,
+  AminoSignJob,
+  AminoSignResponse,
+  AminoSignSpec,
   encodeSecp256k1Signature,
   makeCosmoshubPath,
   OfflineSigner,
-  StdSignDoc,
+  serializeSignDoc,
+  SignMode,
 } from "@cosmjs/launchpad";
-import { serializeSignDoc, SignResponse } from "@cosmjs/launchpad";
 import Transport from "@ledgerhq/hw-transport";
 
 import { LaunchpadLedger, LaunchpadLedgerOptions } from "./launchpadledger";
 
-export class LedgerSigner implements OfflineSigner {
+export class LedgerSigner implements OfflineSigner<AminoSignSpec> {
```

Comment:
Very cool actually how this `OfflineSigner<T>` can set two unrelated variable types in `sign`

Reply:
I'm still not sure how I feel about this, but was inspired by @abefernan here: https://github.com/CosmWasm/dApps/pull/45#discussion_r500124760